src/Splunk.OpenTelemetry.AutoInstrumentation/Logs.cs

```diff
@@ -15,13 +15,17 @@
 // </copyright>
 
 using OpenTelemetry.Logs;
+using Splunk.OpenTelemetry.AutoInstrumentation.Logging;
 
 namespace Splunk.OpenTelemetry.AutoInstrumentation;
 
 internal class Logs
 {
+    private static readonly ILogger Log = OtelLogging.GetLogger();
+
     public void ConfigureLogsOptions(OpenTelemetryLoggerOptions options)
     {
+        ServiceNameWarning.SendOnMissingServiceName(Log);
```

Comment:
What happened if all instrumentation signals are enabled?
Will we have 3 identical records?

Reply:
Yep, I didn't see it as a problem but correct me if I'm wrong. I can add a guard for that.

Reply:
I would like to see it only once. I do not see any reason to log _startup_ warning multiple times.